### PR TITLE
Ensure system agent installer can use private registries

### PR DESCRIPTION
**Pull request by nickgerace**
Relevant issue: https://github.com/rancher/rancher/issues/32403
Relevant comment: https://github.com/rancher/rancher/issues/32403#issuecomment-903128377

### CODE REVIEW

File: pkg/client/generated/management/v3/zz_generated_ecr_credential_plugin.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package client
```
Comment (deniseschannon) [inline pkg/client/generated/management/v3/zz_generated_ecr_credential_plugin.go:1]: why is this part of the PR?

**Review by StrongMonkey — APPROVED** [review]

File: pkg/client/generated/management/v3/zz_generated_ecr_credential_plugin.go (hunk 1 above)
Comment (StrongMonkey) [reply pkg/client/generated/management/v3/zz_generated_ecr_credential_plugin.go:1]: Someone else touched rancher types and forgot to run `go generate` to generate norman types...

**Review by deniseschannon — APPROVED** [review]

File: pkg/client/generated/management/v3/zz_generated_ecr_credential_plugin.go (hunk 1 above)
Comment (deniseschannon) [reply pkg/client/generated/management/v3/zz_generated_ecr_credential_plugin.go:1]: we should find this someone and educate them.

Comment (nickgerace) [reply pkg/client/generated/management/v3/zz_generated_ecr_credential_plugin.go:1]: It's a separate commit for that reason :)

Comment (brendarearden) [reply pkg/client/generated/management/v3/zz_generated_ecr_credential_plugin.go:1]: https://github.com/rancher/rancher/pull/34372/files I did the same on my PR 😆

**Pull request merged**